<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMAGEClass</class>
 <widget class="QMainWindow" name="IMAGEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMAGE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="20,1,40">
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="imagex">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imagey">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>   :   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="prox">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="proy">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>图像二值化</string>
    </property>
    <addaction name="actionGRAY"/>
   </widget>
   <widget class="QMenu" name="menuLaplace">
    <property name="title">
     <string>图像滤波</string>
    </property>
    <addaction name="actionLaplace"/>
    <addaction name="actionGuassian"/>
   </widget>
   <widget class="QMenu" name="menuImage_enhancement">
    <property name="title">
     <string>图像增长</string>
    </property>
    <addaction name="actionImage_linear_enhancement"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>图像分割</string>
    </property>
    <addaction name="actionRegion_growing"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuCalculate"/>
   <addaction name="menuLaplace"/>
   <addaction name="menuImage_enhancement"/>
   <addaction name="menuSegmentation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionGRAY">
   <property name="text">
    <string>RGB2GRAY</string>
   </property>
  </action>
  <action name="actionLaplace">
   <property name="text">
    <string>拉普拉斯锐</string>
   </property>
  </action>
  <action name="actionImage_linear_enhancement">
   <property name="text">
    <string>图像线性增长</string>
   </property>
  </action>
  <action name="actionRegion_growing">
   <property name="text">
    <string>区域生长</string>
   </property>
  </action>
  <action name="actionGuassian">
   <property name="text">
    <string>高斯平滑</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IMAGE.qrc"/>
 </resources>
 <connections/>
</ui>
